trinity/README.md

```diff
@@ -0,0 +1,10 @@
+
+
+
+## Environment Configuration
+
+- `TRINITY_MP_CONTEXT` - The context that new processes will be spawned from the python `multiprocessing` library.
+- `TRINITY_HOME` - Base directory where trinity stores data
+- `TRINITY_CHAIN_DIR` - The base directory where data will be store for the currently running chain.
+- `TRINITY_DATA_DIR` - The directory where the chain data will be stored for the currently running chain.
```

Comment:
I think this means that one of `('TRINITY_CHAIN_DIR', 'TRINITY_DATA_DIR')` contains the in-chain state, and the other contains metadata about the topology of the chain, but wasn't sure which was which.

Reply:
This is piss poor documentation at-the-moment.  What I'm really doing here is keeping a running list of the environment variables so that when the time comes to document them *for-real* we don't have to excavate them from the codebase.